whisk/api.go

```diff
@@ -71,6 +71,32 @@ type Api struct {
     GatewayFullPath string    `json:"gatewayFullPath,omitempty"`
     Swagger         string    `json:"swagger,omitempty"`
     Action          *ApiAction `json:"action,omitempty"`
+    PathParameters   []ApiPathParameter `json:"pathParameters,omitempty"`
+}
+
+type ApiPathParameter struct {
+    Name              string      `json:"name,omitempty"`
```

Comment:
since `name` and `in` values are required, can the `omitempty` be removed from these fields?